### DISCUSSION TITLE
Hopping more data  could be stored in the PAR file

### DISCUSSION THREAD

**Discussion by zhangfengeo** [Anura3D/Anura3D_OpenSource#46] [General]
Hi,there are many useful data of the MP ID are stored in the PAR file,but some data are not include,such as the average strain/stress of one loadstep,which makes user very hard to plot the curve. I suppose the data like this could be stored to help user process data more efficiently.

**Comment by murphy-john** [comment]
You can edit the code to add any output variables you need. We tend to output data in as raw as possible for flexibility in post-processing. 

You could also write a script to average the stresses using the data from the PAR files. 

Alternatively, you can try plotting the results in Paraview.
